.github/workflows/pr.yml

```diff
@@ -1642,4 +1642,70 @@ jobs:
 
     secrets: inherit
     with:
-      version-label: ${{ needs.check-cache.outputs.version-label }}
\ No newline at end of file
+      version-label: ${{ needs.check-cache.outputs.version-label }}
+
+  # This job sets an output `is_publish_build` to `true` if this is a release build
+  check-release-build:
+    runs-on: ubuntu-latest
+    needs: [check-cache]
+    outputs:  
+      is_publish_build: ${{ steps.check_release.outputs.is_publish_build }}
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Get Git tag
+        run: |
+          gitTag=$(git describe --tags --exact-match HEAD) || echo "NONE"
```

Comment:
If the tag does not exist the tag would be "NONE", shouldn't we stop the release if not gittag is defined?

Reply:
If the gittag is `NONE` `is_publish_build` will be `false` in the next step which will skip the release 
<img width="1097" alt="image" src="https://github.com/realm/realm-kotlin/assets/1793238/f2edd7c8-9192-48c8-a947-02d53abea8af">